modules/sophus/1.22.10/patches/add_build_file.patch

```diff
@@ -0,0 +1,39 @@
+--- /dev/null
++++ BUILD.bazel
+@@ -0,0 +1,36 @@
++""" Builds Sophus.
++"""
++
++load("@rules_cc//cc:defs.bzl", "cc_library")
++
++cc_library(
++    name = "sophus",
++    hdrs = [
++        "sophus/average.hpp",
++        "sophus/cartesian.hpp",
++        "sophus/ceres_local_parameterization.hpp",
++        "sophus/ceres_manifold.hpp",
++        "sophus/ceres_typetraits.hpp",
++        "sophus/common.hpp",
++        "sophus/geometry.hpp",
++        "sophus/interpolate.hpp",
++        "sophus/interpolate_details.hpp",
++        "sophus/num_diff.hpp",
++        "sophus/rotation_matrix.hpp",
++        "sophus/rxso2.hpp",
++        "sophus/rxso3.hpp",
++        "sophus/se2.hpp",
++        "sophus/se3.hpp",
++        "sophus/sim2.hpp",
++        "sophus/sim3.hpp",
++        "sophus/sim_details.hpp",
++        "sophus/so2.hpp",
++        "sophus/so3.hpp",
++        "sophus/spline.hpp",
++        "sophus/types.hpp",
++        "sophus/velocities.hpp"
++    ],
```

Comment:
@d-vo Is it possible to add this after the line?
```
includes = ["."],
```
By doing this, developer could use `#include <sophus/...>`. For instance, to include se3.hpp, user can use
```
#include <sophus/se3.hpp>
```
instead of having to use
```
#include "sophus/se3.hpp"
```

Reply:
This is going to be resolved in https://github.com/bazelbuild/bazel-central-registry/pull/1495.